custom-login/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "samples-js-react",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I saw the package.json at bottom of the PR is using `2.0.0`, just wondering what's the difference?

Reply:
Not much of one.  In the past the base level package version would update...sometimes (this sample is only used via git clone, not published to npm, so the version isn't very important) and these packages were left at the 0.1.0 level, as their version is used even less than the base level. 

Since this update included a notable re-write, I elected to bump them as giving SOME information, but the odds that it is even noticed by anyone, much less is useful, are pretty low.